packages/server/lib/util/git.ts

```diff
@@ -0,0 +1,42 @@
+import execa from 'execa'
+
+export interface GitInfo {
+  author: string | null
+  lastModifiedTimestamp: string | null
+  lastModifiedHumanReadable: string | null
+}
+
+// matches <timestamp> <when> <author>
+// output of git log -1 --pretty=format:%ci %ar %an <file>
+// eg '2021-09-14 13:43:19 +1000 2 days ago Lachlan Miller
+
+const regexp = /(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [-+].+?)\s(.+ago)\s(.*)/
+
+export const getGitInfo = (absolutePath: string): GitInfo | null => {
+  try {
+    const gitinfo = execa.sync('git', [
```

Comment:
I don't think this should be `.sync`? We shouldn't be executing anything sync in node